test/variables.yaml.example

```diff
@@ -3,4 +3,5 @@ USHIFT_HOST: microshift
 # USHIFT_USER: User to log into MicroShift's host
 USHIFT_USER: microshift
 # SSH Private key to use when logging into MicroShift's host.
+# Unset this variable to use ssh agent.
 SSH_PRIV_KEY: /home/microshift/.ssh/id_rsa
```

Comment:
Can we use env variable like `$HOME` or `~/.ssh` here instead of `/home/microshift`?